README.md

```diff
@@ -40,3 +40,9 @@ This module can be installed with the `go get` command:
   }
 
 ```
+
+# Exslt
+
+To enable usage of exslt extension, namespace xmlns:exsl="http://exslt.org/common" and it's functions like exsl:node-set(), You'll need to init it once at programm start
+
+    xslt.InitExslt()
```

Comment:
See other comment on changing Go function name to init -- this addition to the readme will no longer be necessary (user won't need to call).